README.md

```diff
@@ -397,99 +517,193 @@ Also if you like to automatically record the execution time of different cells
 [nbextensions](https://jupyter-contrib-nbextensions.readthedocs.io/en/latest/install.html).
 
 A sample notebook is provided at [`dwh/test_spark.ipynb`](dwh/test_spark.ipynb)
-which uses `matplotlib` too, so to run this you need to install it too:
-```
-(venv) $ pip3 install matplotlib
-```
-At the end of this notebook, we wrap up the tests into reusable functions that
-we can use outside the Jupyter environment e.g., for automated indicator
-computation.
+which uses `matplotlib` too, so to run this you need to install it too: `(venv)
+$ pip3 install matplotlib` At the end of this notebook, we wrap up the tests
+into reusable functions that we can use outside the Jupyter environment e.g.,
+for automated indicator computation.
 
+## Integrating with OpenHIM (Middleware component)
 
-# Integrating with OpenHIM (Middleware component)
-For the case, where there are several OpenMRS systems,
-OpenHIM can be used as a middleware component to integrate/track/log requests from several OpenMRS instances.
+For the case, where there are several OpenMRS systems, OpenHIM can be used as a
+middleware component to integrate/track/log requests from several OpenMRS
```

Comment:
This statement is a little bit misleading or I don't fully understand it. I would simply reword this as:

"_The generated FHIR resources can be sent to a middleware like OpenHIM._"

and avoid the topic of multiple OpenMRS instances altogether. Adding @mozzy11 to make sure my suggestion is okay with his intended use-case.

Reply:
Sorry, I should have specified that I only edited the content of the pipeline sections. Happy to update if @mozzy11 confirms the correction in this PR or as a follow-up during the file splits.

Reply:
Sounds good; let's address these changes when we split this and create a separate doc for the OpenHIM case.